tests/php/test_class.jetpack-gutenberg.php

```diff
@@ -129,4 +130,18 @@ function test_plugin_is_not_available_when_not_registered_returns_missing_module
 		$this->assertFalse( $availability['tomato']['available'], 'Availability is not false exists' );
 		$this->assertEquals( $availability['tomato']['unavailable_reason'], 'missing_module', 'unavailable_reason is not "missing_module"'  );
 	}
+
+	function test_get_available_extensions() {
+		$extensions = Jetpack_Gutenberg::get_available_extensions( $this->get_extensions_whitelist() );
```

Comment:
Since that's the default usage, should we also add a test for cases where we don't specify a `$whitelisted_extensions` argument?

Reply:
That's originally how the test was written. But, the list returned from `get_jetpack_gutenberg_extensions_whitelist()` differs between environments. Specifically, that method seems to return an empty array in Travis while returning a proper list of extensions in our local environments. Ideally, we could just mock the return of `get_jetpack_gutenberg_extensions_whitelist()`, but since these are static methods, we'd need to do that by adding a filter to `get_jetpack_gutenberg_extensions_whitelist()` most likely. So, it seemed like adding an argument to short-circuit calling `get_jetpack_gutenberg_extensions_whitelist()` was a good option.